docs/content/reference/install-configuration/entrypoints.md

```diff
@@ -0,0 +1,238 @@
+| `udp.timeout` | Define how long to wait on an idle session before releasing the related resources. <br />The Timeout value must be greater than zero.| 3s (seconds)| No |
+
+<!-- markdownlint-enable MD013 -->
+
+### address
+
+In the Helm Chart, the `entryPoints` `web` (port 80), `websecure` (port 443), 
+`traefik` (port 9000) and `metrics` (port 9100) are created by default.
+The `entryPoints` `web`, `websecure` are exposed by default using a Service.
+
+The default behaviors can be overridden in the Helm Chart.
+
+### asDefault
+
+If there is no `entryPoint` with the `asDefault` option set to `true`, then the 
+list of default `entryPoints` includes all HTTP/TCP `entryPoints`.
+
+If at least one `entryPoint` has the `asDefault` option set to `true`,
+then the list of default `entryPoints` includes only `entryPoints` that have the
+`asDefault` option set to `true`.
+
+Some built-in `entryPoints` are always excluded from the list, namely: `traefik`.
+
+The `asDefault` option has no effect on UDP `entryPoints`.
+When a UDP router does not define the `entryPoints` option, it is attached to all
+available UDP `entryPoints`.
+
+### http.middlewares
+
+- You can attach a list of [middlewares](../../middlewares/http/overview.md)
+to each EntryPoint.
+- The middlewares will take effect only if the rule matches, and before forwarding
+the request to the service.
+- Middlewares are applied in the same order as their declaration.
+- Middlewares are applied by default to every router exposed through the EntryPoint
+(the Middlewares declared on the [IngressRoute](../../routing/routers/index.md#middlewares)
+or the [Ingress](../../routing/providers/kubernetes-ingress.md#on-ingress)
+are applied after the ones declared on the Entrypoint)
+- The option allows attaching a list of middleware using the format 
+`middlewarename@providername` as described in the example below:
+
+```yaml tab="File (YAML)"
+entryPoints:
+  web:
+    address: :80
+    middlewares:
+      - auth@kubernetescrd
+      - strip@file
+```
+
+```yaml tab="Helm Chart Values"
+ports:
+  web:
+    port: :80
+    middlewares:
+      - auth@kubernetescrd
+      - strip@file
+```
+
+```yaml tab="Attach the Middleware"
+# Attach the middleware auth defined using the File provier and the middleware
+# default-prefix defined using a Kubernetes CRD
+traefik.ingress.kubernetes.io/router.middlewares: auth@file,default-prefix@kubernetescrd
+```
+
+!!! warning "Helm Chart"
+    In the Helm Charts, the entryPoints web (port 80), websecure (port 443), 
+    traefik (port 9000) and metrics (port 9100) are created by default. 
+    The entryPoints web, websecure are exposed by default using a Service.
+    The default behaviors can be overriddenin the Helm Chart
+
+### http.encodeQuerySemicolons
```

Comment:
@nmengin @sheddy-traefik Just a thought, we kept the `http.` in the title because our previous documentation documented the option hierarchically. Now I would keep things simple and call this section ecodeQuerySemicolons, wdyt?

Reply:
`ecodeQuerySemicolons` works then, i followed the naming convention in the previous docs, so this chamge makes sense